Docgen/business/plugins/org.polarsys.kitalpha.doc.gen.business.core/src/org/polarsys/kitalpha/doc/gen/business/core/task/IndexingConceptsTask.java

```diff
@@ -508,7 +508,7 @@ private void indexPage(Map<String, IndexItem> indexItems, IFile file)
 		while (mTable.find()) 
 		{
 			for (Entry<String, IndexItem> entry : indexItems.entrySet()) {
-				String currentConcept_html = EscapeChars.forHTML(entry.getValue().getConceptName());
+				String currentConcept_html = entry.getValue().getEscapedForHTMLConceptName();
```

Comment:
is getEscapedForHTMLConceptName() a new method? if yes, where is it located?

Reply:
It is a getter for an attribute of the IndexItem class I have added in the performance optimisation PRs to avoid multiple computations of EscapeChars.forHtml (here and mostly during index computation).